lib/torrent.js

```diff
@@ -88,6 +88,7 @@ export default class Torrent extends EventEmitter {
     this._destroyStoreOnDestroy = opts.destroyStoreOnDestroy || false
     this.store = null
     this.storeOpts = opts.storeOpts
+    this.alwaysChokeSeeders = opts.alwaysChokeSeeders !== undefined ? opts.alwaysChokeSeeders : true;
```

Comment:
This can simply become:
```suggestion
    this.alwaysChokeSeeders = opts.alwaysChokeSeeders ?? true
```

Reply:
Sure I'll change it. I used `var !== undefined` because the same style is being used elsewhere in the code.
e.g. https://github.com/webtorrent/webtorrent/blob/c8ffdda0d0a03571bab2644cb5e0033d47486294/lib/torrent.js#L87C1-L87C91